phpstan.neon

```diff
@@ -7,6 +7,7 @@ parameters:
     treatPhpDocTypesAsCertain: false
     excludes_analyse:
         - 'src/DependencyInjection/Configuration.php'
+        - 'test/functional/Generated/GetGoodImpl.php'

```

Comment:
Is this necessary? Phpstan runs before or after the tests and the GetGoodImpl.php should not exist at that moment